### PR TITLE
feat:created dark mode

### PR DESCRIPTION
**Pull request by KamiyaGaikwad**

### CODE REVIEW

File: styles/docs.css [hunk 1]
```diff
@@ -295,3 +330,65 @@ input::placeholder{
...
+    bottom: 4px;
+    content: "";
+    height: 26px;
+    left: 4px;
+    position: absolute;
+    transition: .4s;
+    width: 26px;
+  }
+  
+  input:checked + .slider {
```
Comment (vinc3nati) [inline styles/docs.css:380]: don't use `input` it's a very general selector and this style will apply to all of the input elements in your code. Instead use a specific class or id to target it.

Comment (vinc3nati) [inline styles/docs.css]: don't use input it's a very general selector and this style will apply to all of the input elements in your code. Instead use a specific class or id to target it.

**Review by vinc3nati — APPROVED** [review]
changes looks good!

**Review by shrey27 — APPROVED** [review]
good to merge

**Pull request merged**